### PR TITLE
D3 graph

### PR DESCRIPTION
**Pull request by ableuler**
Cleaned up version of the latest demo.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -165,10 +162,12 @@ celerybeat-schedule
 
 # dotenv
 
-# virtualenv
-.venv
+# Python virtualenv
```
Comment (leafty) [inline .gitignore]: You should not modify what's being generated by gitignore.io. Check out https://www.gitignore.io/api/node%2Csass%2Cpython%2Cpycharm%2Ball%2Cvisualstudiocode and replace to add .idea files.

File: typings.d.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+declare module '*.json' {
```
Comment (leafty) [inline typings.d.ts]: Copyright header missing here

**Comment by leafty** [comment]
Looks neat. :clap:

File: src/components/graph/icons.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  "resource:file": {
+    "path": "m 24.033898,9.7711864 -7.220339,-6.6864406 -14.4406776,0 c -1.3237289,0 -2.40677971,1.0029661 -2.40677971,2.2288135 l 0,13.3840257 c 0,1.225847 1.08305081,2.217669 2.40677971,2.217669 L 21.627119,20.90411 c 1.323728,0 2.406779,-0.991822 2.406779,-2.217669 l 0,-8.9152546 z m -8.423729,-5.0148305 6.618645,6.1292371 -6.618645,0 0,-6.1292371 z"
+  },
+  "resource:file_version": {
```
Comment (ebiiii) [inline src/components/graph/icons.json]: maybe if you can find a different one for the file versions

Comment (ableuler) [reply src/components/graph/icons.json]: Yeah, I was planning on adding a little "v" on top of the file icon. Wanted to revisit the icons anyway. Maybe we can make that an issue for the future?

Comment (ebiiii) [reply src/components/graph/icons.json]: yep

**Comment by ableuler** [comment]
- Fixed the issues brought up by @leafty
- Rebased the branch and squashed all commits into one as all the intermediate states are broken after removal of the .scss style sheet.

**Comment by ableuler** [comment]
- Fixed the issues brought up by @leafty
- Rebased the branch and squashed all commits into one as all the intermediate states are broken after removal of the .scss style sheet.

**Pull request merged**